data-source.ts

```diff
@@ -0,0 +1,15 @@
+import { DataSource } from 'typeorm';
+
+export const typeOrmConnectionDataSource = new DataSource({
+    type: 'postgres',
+    host: 'localhost',
+    port: 5432,
+    username: 'postgres',
+    password: 'mysecretpassword',
+    database: 'postgres',
+    synchronize: true,
+    logging: false,
+    entities: ['src/**/*.entity.ts'],
+    migrations: ['src/migration/**/*.ts'],
+    subscribers: ['src/subscriber/**/*.ts'],
+});
```

Comment:
Is this still in use? If so we probably wanna make it configurable from env...

Reply:
We need this file to run migrations with the new typeorm api. It's a replacement for this deprecated config file https://github.com/cabinetoffice/gap-find-backend/blob/main/ormconfig.json (will remove). This file on main has the same static values which yeah should come from env really.

Reply:
Have set that up now :)